exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,8 +8,15 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
-
+function calculateGrade(mark) {if (mark >= 80 ){
+     return "A";
+   } else if (mark < 80  && mark > 60 ){
+     return "B";
+   }else if(mark <=60  && mark > 50 ){
```

Comment:
the `mark <=60` check is unnecessary here, as anything <80 >60 will be caught in the above `if` statement